cdp_scrapers/instances/portland.py

```diff
@@ -166,14 +269,10 @@ def get_matter(self, minute_section: Tag) -> Optional[Matter]:
             matter_type = matter_type[1:-1]
 
         # Find result status
-        result_status_element_sibling = minute_section.find(
-            "div", text=re.compile("Disposition"), attrs={"class": "field__label"}
-        )
-        result_status_element = result_status_element_sibling.next_sibling
-        result_status = result_status_element.text
-        if result_status in ["Accepted", "Passed"]:
+        result_status = get_disposition(minute_section)
+        if result_status in ["Accepted", "Passed", "Adopted"]:
```

Comment:
I forgot t update this blob. I must have had low sugar in my brain at this time. Will use the `MATTER*PATTERNS` for cleaner code.